_posts/2018/2018-12-02-write-long-strings-with-wiggly-heredocs.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Write long strings with wiggly HEREDOCs
+description: "If you’re"
+layout: article
+category: ruby
+image:
+  base: '2018/write-long-strings-with-wiggly-heredocs'
+  alt: 'Ball of string'
+  source: 'https://unsplash.com/photos/8VO-UxlJ-Lw'
+  credit: 'Steve Johnson'
+---
+
+There’s multiple ways to write strings in Ruby, but if you are using or generating longer or multi-line strings the `HEREDOC` (or 'Here document') is your friend.
+
+Ruby’s documentation for Strings can be [found here](http://ruby-doc.org/core-2.5.3/doc/syntax/literals_rdoc.html#label-Strings).
```

Comment:
* Remember, this isn't an accessible or semantically correct way of stating the links. You can say: Find out more in [Ruby's documentation for Strings](link here) — or something like that...